types/lexer.d.ts

```diff
@@ -63,6 +63,80 @@ export interface ImportSpecifier {
   readonly a: number;
 }
 
+export interface ExportSpecifier {
+  /**
+   * Exported name
+   *
+   * @example
+   * const source = `export default []`;
+   * const [imports, exports] = parse(source);
+   * exports[0].n;
+   * // Returns "default"
+   *
+   * @example
+   * const source = `export const asdf = 42`;
+   * const [imports, exports] = parse(source);
+   * exports[0].n;
+   * // Returns "asdf"
+   */
+  readonly n: string;
+
+  /**
+   * Local name, or undefined.
+   *
+   * @example
+   * const source = `export default []`;
+   * const [imports, exports] = parse(source);
+   * exports[0].n;
+   * // Returns undefined
+   *
+   * @example
+   * const asdf = 42;
+   * const source = `export { asdf as a }`;
+   * const [imports, exports] = parse(source);
+   * exports[0].n;
+   * // Returns "asdf"
```

Comment:
shouldn't that be an example of `ln`?

so more like

```
   * exports[0].ln;
   * // Returns "a"
```
?

or am I wrong in the assumption that this is the way to get "a"?

Reply:
Yeah that's definitely a typo, thanks for spotting it!